doc/company.texi

```diff
@@ -1243,11 +1247,11 @@ A completion for symbol names typically assists programmers working
 with the source code.
 
 @defun company-capf
-In Emacs' world, the current tendency is to have the completion logic
-executed by @w{@code{completion-at-point-functions}} (@acronym{CAPF})
-implementations.  [Among the other things, this is what the popular
-packages that support language server protocol (@acronym{LSP}) also
-rely on.]
+In the Emacs's world, the current tendency is to have the completion
+logic executed by @w{@code{completion-at-point-functions}}
```

Comment:
"completion logic *provided* by ...", maybe?

Or "supplied by", or "implemented by".

Reply:
Yep, "provided by" should be fine.